Authenticator/Source/AppViewController.swift

```diff
@@ -82,6 +82,13 @@ extension AppViewController: ActionHandler {
 
 extension AppViewController: AppPresenter {
     func updateWithViewModel(viewModel: AppViewModel) {
+        updateWithViewModel(viewModel, ephemeralTokenListMessage: nil)
+    }
+
+    func updateWithViewModel(viewModel: AppViewModel, ephemeralTokenListMessage: EphemeralMessage?) {
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 101 characters (line_length)